swagger_to_sdk_config.json

```diff
@@ -0,0 +1,101 @@
+{
+  "meta": {
+    "version": "0.1.0",
+    "language": "NodeJS",
+    "autorest": "latest"
+  },
+  "data": {
+    "arm-authorization/2015-07-01/swagger/authorization.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 1 -Modeler Swagger",
+      "output_dir": "lib/services/authorizationManagement/lib"
+    },
+    "arm-batch/2015-12-01/swagger/BatchManagement.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 1 -Modeler Swagger",
+      "output_dir": "lib/services/batchManagement/lib"
+    },
+    "batch/2016-02-01.3.0/swagger/BatchService.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 1 -Modeler Swagger",
+      "output_dir": "lib/services/batch/lib"
+    },
+    "arm-cdn/2015-06-01/swagger/cdn.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 2 -Modeler Swagger",
+      "output_dir": "lib/services/cdnManagement/lib"
+    },
+    "arm-compute/compositeComputeClient.json": {
+      "autorest_options": "-header MICROSOFT_MIT_NO_VERSION -FT 1 -Modeler CompositeSwagger",
+      "output_dir": "lib/services/computeManagement2/lib"
```

Comment:
@lmazuel - Compute uses CompositeSwaggerModeler. I hope you have taken this into consideration.